packages/kotti-ui/source/kotti-avatar/KtAvatar.vue

```diff
@@ -129,19 +128,22 @@ export default defineComponent<KottiAvatar.PropsInternal>({
 
 	&--is-size-small {
 		width: 1.6rem;
-		height: 1.6rem;
+		max-height: 1.6rem;
```

Comment:
Why remove `width`? with long content, it'll shrink  the image. Maybe a `min-width` is missing?

Reply:
`width` is not removed, I just changed `height` for `max-height` and add the `aspect-ratio: 1/1`